analyses/mwi/dryspells_trigger/exploration/explore_multiple_dry_spells.md

```diff
@@ -0,0 +1,36 @@
+## Explore multiple dry spells in a season
+
+Simple and quick code to look at times where there have been multiple dry spells in a season
+
+```python
+import pandas as pd
+import os
+
+dd = os.environ["AA_DATA_DIR"]
+
+df = pd.read_csv(os.path.join(dd, 'public', 'processed', 'mwi', 'arc2', 'mwi_arc2_dry_spells_2000_2020.csv'),
+                 parse_dates = ['dry_spell_first_date', 'dry_spell_last_date'])
+df = df[df.during_rainy_season]
+```
+
+Let's first just look at the times when we have multiple dry spells for an ADM2 area in the same season.
+
+```python
+df[df.duplicated(subset=['ADM2_PCODE', 'season_approx'], keep=False)]
+```
+
+We can see from above that we have 6 instances of this occurring. However, if we look at the second recorded dry spell, only 1 of them occurs within our monitoring window. The rest are observed in March after the monitoring has stopped.
+
+```python
+df[df.duplicated(subset=['ADM2_PCODE', 'season_approx'], keep='first') & (df.dry_spell_first_date.dt.month <= 2) & (df.dry_spell_first_date.dt.day <= 22)]
+```
+
+However, in this case, the first dry spell detected was at the beginning of November, again falling outside of the monitoring window.
+
+```python
+df[df.duplicated(subset=['ADM2_PCODE', 'season_approx'], keep='last') & (df.dry_spell_last_date.dt.month >= 11)]
+```
+
+Thus, none of these would actually change the return period for our analysis based on the monitoring window set up for the V1 Malawi trigger. However, it's still something we need to consider when implementing the monitoring.
+
+Given the focus on geographical spread of a dry spell to measure impact rather than duration, I think makes sense not to count a district twice if 2 separate dry spells occur. Given our short time window for monitoring, these dry spells are likely only broken apart by a small number of days of rainfall. Since the impact of a dry spell is already sufficiently high when the 14 days have passed, the marginal impact of a second dry spell is likely less. The same intuition is why we wouldn’t double count a dry spell that last for 30 days, for instance. However, important to raise for discussion and confirming our decision
```

Comment:
Eventually we'll need to look at number of days between dry spells within an adm2 more closely. We've seen 1-2 days "break up" a very long dry spell and end up creating two shorter dry spells in our datasets. I agree with you that those shouldn't be considered tow separate events and count towards the threshold for the observational trigger V1. On the other hand, if they were separated by 7+ days, or perhaps the 2 dry spells did not really overlap within an adm2, I'd consider those events separate. 

I'm okay with not counting a dry spell in the same adm2 in Jan and Feb 2022 twice (ie for V1). But let's keep this conversation in mind as we do the monitoring to make sure the "unlikely, does not need to be resolved perfectly yet" does not become overlooked or not considered carefully if it were to occur during the monitoring. Also: let's add a deeper dive into these edge cases for V2.